### ISSUE TITLE
Question about 'kidney_align.draw_3D'

### ISSUE THREAD

**Issue by Changzhanhe** [gao-lab/SLAT#3]
What a great tool for spatial slides alignment! :-) 

I have met an error when I used draw_3D function to plot cell type alignment between two slides, an error occurred:

_"TypeError: draw_lines() takes from 4 to 6 positional arguments but 7 were given."_

Hope for your reply. Thanks!

**Comment by xiachenrui** [comment]
Thanks for your interest and feedback !

 I can not reproduce your error in my computer (with latest version of SLAT). Could you update to the latest version (0.2.1) following our [tutorial](https://github.com/gao-lab/SLAT#pypi). If updating can not solve your error, counld you please provide your codes?

Best!
Chen-Rui

**Comment by xiachenrui** [comment]
We close this issue because key information is missing. If you need more help, please open it again.

**Comment by Starlitnightly** [comment]
Hi, @xiachenrui 

After testing, this bug is caused by a function parameter error in version 0.2.1 on pypi, which is not reported if installed version 0.2.2 using the github repository.

Also, `scslat 0.2.2 requires scipy<1.9,>1.3` seems to be an unnecessary limitation, I run scslat on scipy==1.11.4 and everything works fine.

Also, some of the arguments to the functions in the tutorial don't seem to work correctly when the version of scslat is 0.2.2. For example:

```
kidney_align = match_3D_celltype(adata1_df, adata2_df, matching, meta='celltype', highlight_celltype = [[''],['']],
                                 subsample_size=300, highlight_line = ['blue'], scale_coordinate = True )
```
    ValueError: ['blue'] is not a valid value for colour

Sincerely,

Zehua

**Comment by lengfei5** [comment]
Many thanks @Starlitnightly , I have gotten the same issue with v0.2.1 and solved it by updating to v0.2.2.
